packages/@react-spectrum/dialog/docs/AlertDialog.mdx

```diff
@@ -0,0 +1,176 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/dialog';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ActionButton} from '@react-spectrum/button';
+import {AlertDialog, DialogTrigger} from '@react-spectrum/dialog';
+```
+
+# AlertDialog
+
+<p>{docs.exports.AlertDialog.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['AlertDialog']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/dialog/#Options'}
+  ]} />
+
+## Example
+
+```tsx example
+<DialogTrigger>
+  <ActionButton>Save</ActionButton>
+  <AlertDialog title="Low Disk Space" variant="warning" primaryLabel="Confirm">
+    You are running low on disk space. Delete unnecessary files to free up space.
+  </AlertDialog>
+</DialogTrigger>
+```
+
+## Content
+
+Unlike Dialog, `slots` are not supported in AlertDialog since its layout is very deliberate.
+The body of an AlertDialog can be provided by passing children to the component. The AlertDialog also 
+supports three optional buttons in its footer: a confirmation button, a secondary button, and a cancel button.
```

Comment:
Actually this is a bug. The primary button (primaryLabel) is required because you cannot have a dismissible AlertDialog. 
Can you update the type please?
https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-types/dialog/src/index.d.ts#L58